.github/workflows/test_suite.yml

```diff
@@ -0,0 +1,36 @@
+name: Test Suite
+
+on: [push]
+
+jobs:
+  test:
+    runs-on: ubuntu-20.04
+    strategy:
```

Comment:
Please check https://github.com/Block-Logic/solana-rpc-ruby/blob/main/.github/workflows/specs.yml
It would be nice to check the gem against multiple ruby versions